src/pages/evm/wallet/ReceivePage.vue

```diff
@@ -1,21 +1,151 @@
 <script lang="ts">
 import { defineComponent } from 'vue';
 import AppPage from 'components/evm/AppPage.vue';
+import AddressQR from 'components/evm/AddressQR.vue';
+import UserInfo from 'components/evm/UserInfo.vue';
+import { getAntelope, useAccountStore } from 'src/antelope';
+
+const accountStore = useAccountStore();
+const ant = getAntelope();
 
 export default defineComponent({
     name: 'ReceivePage',
     components: {
         AppPage,
+        AddressQR,
+        UserInfo,
+    },
+    methods: {
+        goBack() {
+            this.$router.back();
+        },
+        copyToClipboard() {
+            navigator.clipboard.writeText(this.address);
+            ant.config.notifySuccessHandler(this.$t('settings.copied_ok'));
+        },
+    },
+    computed: {
+        loggedAccount() {
+            return accountStore.loggedEvmAccount;
+        },
+        address() {
+            return accountStore.loggedEvmAccount?.address;
+        },
+        isMobile() {
+            return this.$q.screen.lt.md;
+        },
     },
 });
 </script>
 
 <template>
 <AppPage>
-    <p class="text-black">Receive page</p>
+    <template v-slot:header>
+        <div class="c-receive-page__title-container">
+            <p class="c-receive-page__title"> {{ $t('evm_wallet.receive') }}</p>
+            <p class="c-receive-page__subtitle"> {{ $t('evm_wallet.scan_qr') }}</p>
+            <AddressQR :address="address" class="c-receive-page__qr-code" />
+        </div>
+        <q-btn
+            class="c-receive-page__back-button"
+            flat
+            dense
+            label="Back"
+            icon="arrow_back_ios"
+            @click="goBack"
+        />
+    </template>
+
+    <div
+        :class="{
+            'c-receive-page__user-info-container': true,
+            'c-receive-page__user-info-container--mobile': isMobile,
+        }"
+    >
+        <UserInfo
+            class="c-receive-page__user-info"
+            :displayFullAddress="true"
+            :showCopyBtn="!isMobile"
+            :showUserMenu="false"
+            :account="loggedAccount"
+        />
+
+        <q-btn
+            v-if="isMobile"
+            color="primary"
+            :label="$t('evm_wallet.copy')"
+            class="wallet-btn c-receive-page__copy-btn"
+            @click="copyToClipboard"
+        />
+    </div>
+
 </AppPage>
 </template>
 
 <style lang="scss">
+.q-btn.wallet-btn {
+    width: auto;
+    padding: 13px 24px;
+    font-size: 0.9rem;
+    font-weight: 600;
+    text-transform: uppercase;
+    &+& {
+        margin-left: 16px;
+    }
+}
```

Comment:
This piece of styling code should not be here. Instead, it should be present in a global place to affect all `q-btn.wallet-btn` buttons on the web app.

@karynemayer and @EJSG, do we have a styling solution for all buttons on the app? Should we create a separate issue to address this? I wait your opinions

Reply:
The updated styles would be the buttons with the accent color as a background and text only. The hover, click, etc should follow quasar's defaults. 

The gradient button is only being kept for legacy reasons

Reply:
I [just mention](https://github.com/telosnetwork/telos-wallet/issues/259#issuecomment-1522360236) your feedback about this button styling on the [issue](https://github.com/telosnetwork/telos-wallet/issues/259) of centralizing all texts and button styles.